### PR TITLE
add canvas resize module

### PR DESCRIPTION
**Pull request by tech4GT**
cc @jywarren 
image_0
final canvas size: 1000 x 1000
x: 500, y: 500
image_1

### CODE REVIEW

File: src/Modules.js [hunk 1]
```diff
@@ -31,12 +32,12 @@ module.exports = {
...
-  'replace-color':require('./modules/ReplaceColor'),
+  'replace-color': require('./modules/ReplaceColor'),
   'resize': require('./modules/Resize'),
   'rotate': require('./modules/Rotate'),
   'saturation': require('./modules/Saturation'),
   'text-overlay': require('./modules/TextOverlay'),
   'threshold': require('./modules/Threshold'),
   'tint': require('./modules/Tint'),
-  'color-temperature': require('./modules/ColorTemperature')
+  'canvas-resize': require('./modules/CanvasResize')
```
Comment (Harshithpabbati) [inline src/Modules.js]: @tech4GT may be you can add it in alpabetical order. So that the list will not become unordered.

Comment (tech4GT) [reply src/Modules.js]: Hi @Harshithpabbati Aren't we sorting these anyway?
I mean earlier the color-temperature was way at the end but it shows up just fine in the UI.

Comment (Harshithpabbati) [reply src/Modules.js]: Oh yeah I thought we weren't sorting it. Sorry.

**Comment by jywarren** [comment]
Hi! Can we add a test for this, as I think we really want good tests for
all modules at this point? Thanks so much!!!!

**Comment by HarshKhandeparkar** [comment]
Will this only be available in the browser?

**Comment by tech4GT** [comment]
No harsh, works in node.

**Comment by HarshKhandeparkar** [comment]
Why is it called canvas resize?

**Comment by tech4GT** [comment]
@jywarren tests added!

**Comment by jywarren** [comment]
Oh haha I guess in the sense of a canvas more generally, but maybe we
should clarify in the docs :-)

**Comment by jywarren** [comment]
Poor Mario! 😂

**Comment by jywarren** [comment]
Awesome!!!

**Comment by tech4GT** [comment]
@jywarren haha, what should we take up next?

**Comment by jywarren** [comment]
I guess geotiff output? This may need to be a special "output module" as
we've discussed in another issue... I forget where! A type of module that
doesn't produce something chainable in our system, you know?

**Comment by jywarren** [comment]
We had discussed putting gif generation into such a module type. What do
you think?

**Comment by HarshKhandeparkar** [comment]
How about a bg-color for the canvas?

**Comment by jywarren** [comment]
That'd be a good follow-up! Thanks!!

**Comment by niravasher** [comment]
This is on my GSoC proposal, I want to work on this

**Comment by HarshKhandeparkar** [comment]
Another idea, instead of absolute sizes, relative sizes like `5x` or `500%` can be used or can be made optional, either absolute or relative.

**Comment by niravasher** [comment]
This is on my GSoC proposal, I want to make the changes, please

**Comment by HarshKhandeparkar** [comment]
I was just giving ideas. Anybody can implement those.

**Pull request merged**